solutions/instances/catalogValidationValues.json.template

```diff
@@ -0,0 +1,5 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "resource_group_name": $PREFIX,
+  "existing_kms_guid": $HPCS_US_SOUTH_GUID
```

Comment:
Suggest adding kms_region here too since our instance is in us-south:
`"kms_region": "us-south"`